src/nvim/lua/fs.c

```diff
@@ -0,0 +1,31 @@
+#include <assert.h>
+#include <lauxlib.h>
+#include <stdlib.h>
+
+#include "nvim/lua/fs.h"
+#include "nvim/vim_defs.h"
+
+#ifdef MSWIN
+// uncrustify:off
+# include <windows.h>  // NOLINT(llvm-include-order)
+# include <fileapi.h>
+// uncrustify:on
+#endif
+
+/// Get the current directory of a drive in Windows.
+int fslua_get_drive_cwd(lua_State *L)
```

Comment:
Any chance `vim.uv.fs_realpath` avoids the need for this?

Reply:
Oh hey that actually does work. Thanks a lot for the suggestion